### PR TITLE
Menu list and about page implementation

### PR DESCRIPTION
**Pull request by benshidanny11**
- Implemented the about page.
- Implemented the popup menu for links
- Implemented show and hide menu using javascript

- _**Project demonstration link: [Click to open](https://www.loom.com/share/db635277ceec4778b45349bd894a2a56)**_

### CODE REVIEW

File: assets/styles/css/about.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@media only screen and (max-width: 768px) {
+  *,
+  ::after,
+  ::before {
+    box-sizing: border-box;
+  }
+
+  .about-title {
+    font-size: 42px;
+  }
```
Comment (george-shammar) [inline assets/styles/css/about.css:50]: - [ ] Your pages should look almost identical to the original design, outside the content differences. There are marked differences between your design and the template. Especially in terms of margins and borders e.t.c. Your images appear larger than the normal template. Please, let's make them identical.

File: assets/styles/css/main.css [hunk 2]
```diff
@@ -211,6 +216,62 @@ body {
...
+  background: var(--color-main);
+}
+
+.show-or-hide {
+  visibility: hidden;
+}
+
 @media only screen and (max-width: 768px) {
   *,
   ::after,
```
Comment (george-shammar) [inline assets/styles/css/main.css:277]: - [ ]  The CSS styling should be built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around. Here and in other parts of your css code, you have not developed with "mobile first" approach. Please, let's correct it.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @benshidanny11 , 

Good job so far!

:heavy_check_mark: You have put in good effort and this is noteworthy. Good job :+1: 
:heavy_check_mark: Your repository looks professional. :clap: 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Not Applicable

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @benshidanny11  🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️ 🚧 
_Check the comments under the review._

- [ ] Kindly update your `README` file to properly **Acknowledge** [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — APPROVED** [review]
Hi @benshidanny11 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                and encouragement to the Rwandan Artists and entertainers.
+              </p>
+            </div>
+          </div>
+        </div>
+      </footer>
+    </main>
+    <script src="js/main.js"></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"
```
[the remark this answers is not in the document]
Comment (benshidanny11) [reply about.html]: Hello @george-shammar , Thanks for giving me the feedback. I am going to work on it.

File: README.md [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 ## Live demo
 
-[Click here to visit]()
+[Click here to visit](https://benshidanny11.github.io/consert_page/)
 
 
 ## Getting Started
```
[the remark this answers is not in the document]
Comment (benshidanny11) [reply README.md:19]: Hello @george-shammar , Thanks for giving me the feedback. I am going to work on it.

File: assets/styles/css/main.css (hunk 2 above)
Comment (benshidanny11) [reply assets/styles/css/main.css]: Hello @george-shammar , Thanks for giving me the feedback. I am going to work on it.

**Comment by benshidanny11** [comment]
### Hi @AshakaE ✋,

Thank you so much for approving my project 🙏

**Pull request merged**